RCS file: /cvsroot/freeimage/FreeImage/Source/LibJXR/image/sys/strcodec.h,v
retrieving revision 1.3
diff -u -r1.3 strcodec.h
--- FreeImage/Source/LibJXR/image/sys/strcodec.h	20 Feb 2015 00:35:45 -0000	1.3
+++ FreeImage/Source/LibJXR/image/sys/strcodec.h	22 Jul 2016 22:56:30 -0000
@@ -65,6 +65,12 @@
 #include "../x86/x86.h"
 #endif
 
+#if defined(MACOSX)
+#define PACKETLENGTH (1U<<12)   // 4kB
+#define UINTPTR_T uintptr_t
+#define INTPTR_T intptr_t
+#endif
+
 #ifndef UNREFERENCED_PARAMETER
 #define UNREFERENCED_PARAMETER(P) { (P) = (P); }
 #endif // UNREFERENCED_PARAMETER
